hack/test

```diff
@@ -60,11 +63,24 @@ testReportsVol="-v $testReportsDir:/testreports"
 gotestsumArgs="--format=standard-verbose --jsonfile=/testreports/go-test-report$TEST_REPORT_SUFFIX.json --junitfile=/testreports/junit-report$TEST_REPORT_SUFFIX.xml"
 gotestArgs="-mod=vendor -coverprofile=/testreports/coverage-report$TEST_REPORT_SUFFIX.txt -covermode=atomic"
 
+if [[ "$GOBUILDFLAGS" == *"-race"* ]]; then
+  if [ "$CGO_ENABLED" != "1" ]; then
+    echo >&2 "go race detector requires CGO_ENABLED=1"
+    exit 1
+  fi
+  # force buildkitd to halt on detected race conditions, which will cause the tests to fail
+  export GORACE="halt_on_error=1"
+  export VERIFYFLAGS="" # prevent -static verification
```

Comment:
Don't think it's needed, xx is only used in the Dockerfile to build BuildKit and the integration tests don't build builkitd.